nanodbc/nanodbc.cpp

```diff
@@ -3357,9 +3390,7 @@ connection::connection(const connection& rhs)
 }
 
 #ifndef NANODBC_NO_MOVE_CTOR
-connection::connection(connection&& rhs) NANODBC_NOEXCEPT : impl_(std::move(rhs.impl_))
-{
-}
+connection::connection(connection&& rhs) NANODBC_NOEXCEPT : impl_(std::move(rhs.impl_)) {}
```

Comment:
Unless you did run clang-format, this looks like an accidental change and should not be mixed in.